### ISSUE TITLE
Health checks persist after successful deployment

### ISSUE THREAD

**Issue by olalonde** [deis/deis#4968]
I enabled health checks with `deis config:set HEALTHCHECK_URL=/_health` for an app and once a deploy successfully completes (the health check endpoint returns a 200 response), I am still seeing periodic (10s) requests on `/_health` from `Go-http-client/1.1`. Aren't health checks supposed to stop once one of them passed?

**Comment by bacongobbler** [comment]
Nope! health checks occur continuously to ensure the health of the container. If it fails, publisher will remove that container from etcd, effectively removing it from the router.

**Comment by olalonde** [comment]
Oh right, makes sense thanks.

**Comment by olalonde** [comment]
I should have tested this more carefully but I'll write down my story here for posterity.

I setup a health check on my app which only returns a 200 status code if the database is reachable. 

Unfortunately, on Friday evening, the AWS machine running our database had a hardware failure which caused the database to go down for about 20 minutes. In the meantime, the health check started failing and our app was eventually completely removed from the router, making it display an Nginx error page. When the database eventually went back up, our app was still down because it had failed to many health checks I presume and I had to manually boot it back up.

I have now removed the health check to prevent this sort of situation from happening.
